speedy-recovery-client/src/components/secured/home/HomePageForParent.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+
+class HomePageForParent extends Component {
+  static defaultProps = {
+    event: {
+      start: [],
+      patient: [],
+      user: []
+    }
+  };
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      appointmentTime: {},
+      user: {},
+      patient: {}
+    };
+  }
+
+  componentWillMount() {
+    setTimeout(() => {
```

Comment:
What is this?? Please explain :(
1. why are you waiting a second before rendering?
2. Please don't copy the props into the state! Instead use the variables from props everywhere in this component